packages/react-search-ui-views/src/__tests__/BooleanFacet.test.js

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+import BooleanFacet from "../BooleanFacet";
+import { shallow } from "enzyme";
+
+const params = {
+  label: "A Facet",
+  onRemove: jest.fn(),
+  onChange: jest.fn(),
+  options: [
+    {
+      value: "true",
+      count: 10
+    }
+  ],
+  values: []
+};
+
+it("renders", () => {
+  const wrapper = shallow(<BooleanFacet {...params} />);
+  expect(wrapper).toMatchSnapshot();
+});
+
+it("renders with className prop applied", () => {
+  const customClassName = "test-class";
+  const wrapper = shallow(
+    <BooleanFacet className={customClassName} {...params} />
+  );
+  const { className } = wrapper.props();
+  expect(className.includes(customClassName)).toBe(true);
+});
```

Comment:
Thanks for adding tests! Could you also add a check that onChange or onRemove are called depending on isSelected value?